pkg/k8s/factory_functions.go

```diff
@@ -471,6 +471,7 @@ func ConvertToNode(obj interface{}) interface{} {
 			ObjectMeta:      concreteObj.ObjectMeta,
 			StatusAddresses: concreteObj.Status.Addresses,
 			SpecPodCIDR:     concreteObj.Spec.PodCIDR,
+			SpecPodCIDRs:    concreteObj.Spec.PodCIDRs,
```

Comment:
How will this work in the cases of where `IPv4Range` and `IPv6Range` are set in the configuration for Cilium? Should some type of precedence be established?

Reply:
Same as it works today where `IPv4Range` is set in the the configuration for Cilium?

Reply:
I must be missing something then :/ so there is no overlap between the aforementioned `IPv*Range` variables and the PodCIDRs which are provided here? These CIDRs are for node events, yes, but it looks like we also provide a way to configure the pod CIDRs via our CLI. What do those variables do then vs. the PodCIDRs being allocated by K8s for a given node?

Reply:
The CLI flags overwrite the pod CIDRs from k8s node spec.